src/components/Slot/Slot.tsx

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react'
+import * as _ from 'lodash'
```

Comment:
Please remove this import as it is not used.

Reply:
you are both right. in the meantime though I removed the code from Slot that used `invoke` (used that for the Ref prop callback). Will remove lodash.